### DISCUSSION TITLE
Which synths do I lose if I install Hacktribe on electribe synth?

### DISCUSSION THREAD

**Discussion by SwannBra** [bangcorrupt/hacktribe#234] [Q&A] [answered]
Which synth do i loose if i instal hacktribe on my electribe?

**Comment by AndrewSosna** [comment, marked as answer]
Depending on what Electribe you have.
In case of Synth version you lose PCM oscillators, in other words - sampled instruments like Piano, Sax and so on, but you get an ability to use load your own samples, just like on E2 Sampler. 
I don't have E2 Sampler and can't confirm, but it looks like you don't lose anything on it.

**Reply by SwannBra** [reply]
Ok, thank you. I have the synth version..
 I have already made a lot of sounds that i use in my songs, and i didn't made it with sampled sounds. So are you telling me that i won't loose my sounds? With the same filters, same modulations and the same IFX?

**Reply by AndrewSosna** [reply]
I'm not near my E2, can't confirm, but I think the order of the oscillators may be different, so some of your parts will sound different, some will not sound at all. I'd say if you switch from E2 to Hacktribe I'd recommend kind of starting over, use a different SD Card or back it up and erase it.

Anyways, you can always return to the original version if you don't like it.
Just be careful, for E2 Synth you need to modify the header of the firmware, see the instructions. And when you're restoring it back your synth will think that it's Sampler now, so to restore you will need an original SYNTH firmware, but with a modified header so it looks like a Sampler one. See the instructions and discussions here.

**Reply by SwannBra** [reply]
Ok, Thank you very much!

**Reply by AndrewSosna** [reply]
Np, but again, be extra careful, follow the instructions. It's safe, but you must be sure that the firmware you're applying is correctly hacked. Make sure that CRC is correct after applying the patch, and if it's not - DO NOT update to this firmware. Otherwise you may get a brick.
